test/e2e/logs_test.go

```diff
@@ -553,7 +556,6 @@ var _ = Describe("Podman logs", func() {
 
 	It("podman pod logs with container names", func() {
 		SkipIfRemote("Remote can only process one container at a time")
-		SkipIfInContainer("journalctl inside a container doesn't work correctly")
```

Comment:
Here you removed, the SkipIfInContainer() but in other places you kept it. I think we can remove it in all places in this file with your change. It should be consistent at the very least.